newIDE/app/scripts/extract-changelog.js

```diff
@@ -199,4 +202,11 @@ const formatHiddenCommitMessage = commit => {
 
   shell.echo(`\n## 🐛 Bug fixes\n`);
   shell.echo(fixCommits.map(formatCommitMessage).join('\n'));
+
+  if (devCommits) {
+    shell.echo(`\n<details>\n`);
+    shell.echo(`\n<summary> 🛠 Developers</summary>\n`);
```

Comment:
We'll have to test how this renders inside GDevelop when showing the changelog

(if you want to try, launch Storybook and search the `Changelog` component. Modify the code in `GDevelopServicesTestData.js` to insert this and see how it renders)